clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -3809,6 +3836,15 @@ raw2trace_t::accumulate_to_statistic(raw2trace_thread_data_t *tdata,
     case RAW2TRACE_STAT_SYSCALL_TRACES_DECODED:
         tdata->syscall_traces_decoded += value;
         break;
+    case RAW2TRACE_STAT_SYSCALL_TRACES_DECODE_FAILED:
+        tdata->syscall_traces_decode_failed += value;
+        break;
+    case RAW2TRACE_STAT_SYSCALL_TRACES_DECODE_RECOVERABLE_ERROR_COUNT:
+        tdata->syscall_traces_decode_recoverable_error_count += value;
+        break;
+    case RAW2TRACE_STAT_SYSCALL_TRACES_DECODE_EMPTY:
+        tdata->syscall_traces_decode_empty += value;
```

Comment:
Maybe this one too: s/decode/conversion/

Reply:
Done